### DISCUSSION TITLE
Display: grid in Webstudio

### DISCUSSION THREAD

**Discussion by mmm777** [webstudio-is/webstudio-community#100] [General]
Hello, is it possible to implement a grid in Webstudio? In Webflow, it's a very handy tool.

**Comment by kof** [comment]
Currently only with custom CSS over HTML Embed

**Comment by bryanbemis** [comment]
These instructions worked for me: [How to use grid display in Webstudio?](https://apps.webstudio.is/builder/94d4eaf9-6996-4143-9af1-0b8bb5cce8eb?authToken=63aa7fe0-5c56-4768-9962-5e733b29aad6&mode=preview).

Some tips:
1. After entering your code in the HTML Embed component, make sure you turn on the "Show" and "Run script on canvas" switches there or you'll wonder why it isn't working.
![image](https://github.com/webstudio-is/webstudio-community/assets/8339546/5a7bc7c3-0c17-4fe4-8222-25fd2694a967)

2. You'll need to apply the grid class name you defined in the HTML Embed to the grid wrapper component. Do not add this as a design token under Style > Style Sources as usual. Instead, you need to add the class name in **Settings > Properties > Class Name**. Also, make sure the "Show" switch is turned on.
![image](https://github.com/webstudio-is/webstudio-community/assets/8339546/5f72ed9c-e043-42f1-ab96-2926472778cf)
